en/docs/develop/carbon-remote-user-store-manager.md

```diff
@@ -42,3 +42,8 @@ following steps.
 
 3.  Update other fields as required (description is give for each
     property) and click **Add**.
+
+4. Import the public certificate of the remote server to primary
+   IS(IS1)'s
+   `<IS_HOME>/repository/resources/security/client-truststore.jks` file
+   and restart the server.
```

Comment:
... to the primary WSO2 Identity Server instance's (IS1) truststore (`client-truststore.jks`) that is in the `<IS_HOME>/repository/resources/security` directory and restart WSO2 Identity Server.